src/checks/generic/json-file-extension.ts

```diff
@@ -0,0 +1,22 @@
+import { basename, extname } from 'path';
+import { exit } from 'process';
+import { Check } from '../../types/checks';
+
+const check: Check = {
+    id: 'json-file-extension',
+    desc: 'Data files should end with `.json` (#2025)',
+    url: 'https://github.com/datenanfragen/data/#data-formats',
+    severity: 'ERROR',
+    run: (json, ctx) => {
+        const filename = basename(ctx.file_path); 
+        const file_extension = extname(ctx.file_path)
+        
+        if (file_extension != ".json")
+            return {
+                message: `File \`${filename}\` should end in .json. Please rename the file.`,
+                location: { start: { line: 1 } },
```

Comment:
This won't make much sense as suggestions from the reviewbot, but afaik there is no way for a code suggestion to rename a file in GH. @baltpeter do you have an opinion on this?

Reply:
> afaik there is no way for a code suggestion to rename a file in GH.

No, there isn't. I agree, this should just be a comment without a suggestion.